spec/features/hub/filtered_clients_bulk_action_spec.rb

```diff
@@ -35,7 +35,7 @@
     perform_enqueued_jobs
     visit current_path
 
-    expect(page).to have_text "Successful Bulk Client Organization Update"
-    expect(page).to have_text "You successfully moved 30 clients to Rooster Brew."
+    expect(page).to have_text("Successful Bulk Client Organization Update", wait: 5)
```

Comment:
this test is a little flakey sometimes because it doesn't allot enough time for the job to go through to get a success message. Adding this will allow for the test to wait up to 5 seconds to look for the text before failing

Reply:
oof 5 seconds is kind of long, can it be shorter, or is 5 seconds what we need? and do we need to wait for both expect for up to 5 seconds, or can we just put it on this one & remove from the second?

Reply:
oops posted this below but posting here as well for continuity:
@arinchoi03 yes I agree 5 seconds is long but I just doubled checked and this method will only wait up to five seconds if it can't find the text, if it finds the text in less than 5 seconds it will move on. I agree that its also potentially a long time before failing the test if the test should truly fail but I still think it will save more time in the long run from not having to re-run the whole test suite if it fails incorrectly.